CONTRIBUTING.md

```diff
@@ -24,42 +24,42 @@ The following is a set of guidelines for contributing to this repository.
 Anyone is welcome to propose changes to the contents of this repository by creating a new [Issue](https://github.com/eiffel-community/eiffel-gerrit-plugin/issues) ticket in GitHub. These requests may concern anything contained in the repo: changes to documentation, bug fixes, requests for additional information, additional event types, requests for additional examples, new featuers et cetera.
 
 When posting a new issue, try to be as precise as possible and phrase your arguments for your request carefully. Keep in mind that collaborating on software development is often an exercise in finding workable compromises between multiple and often conflicting needs; this is particularly true for defining a shared protocol such as Eiffel. In particular, pay attention to the following:
-1. What type of change is requested?
-1. Why would you like to see this change?
-1. Can you provide any concrete examples?
-1. Which arguments in favor can you think of?
-1. Which arguments against can you think of, and why are they outweighed by the arguments in favor?
+1.  What type of change is requested?
```

Comment:
How about making a PR to the community including these changes. Should it be in the template repository I guess? So we can cascade these changes to all repositories in the eiffel community?

Reply:
Sure great idea, we can raise an issue in community and can also raise a PR there to fix.

Reply:
I will do that

Reply:
https://github.com/eiffel-community/eiffel-repository-template/pull/3